querybook/server/lib/metastore/base_metastore_loader.py

```diff
@@ -534,9 +534,9 @@ def get_table_and_columns(
         """
         pass
 
-    def get_data_element(self, data_element_name: str) -> DataElementTuple:
+    def get_data_element(self, data_element_name: str) -> Optional[DataElementTuple]:
```

Comment:
@abstractmethod is also needed ?

Reply:
didn't add AbstractMethod on purpose, for people they dont have data element, they dont have to implement this function.